stackable-base/Dockerfile

```diff
@@ -1,5 +1,5 @@
-# 8.9-1029 as of 2023-12-01
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9@sha256:0e1879febd1be94e0d234060770ce33a543488374d91881259de9119ea75d33f
+# 8.8-1014 as of 2023-07-20
+FROM registry.access.redhat.com/ubi8@sha256:449da7f8f2ef6285a8445a1e31af57a97b9dae5dcf009b1629c59742c89c68c3 
```

Comment:
This is copy / paste / merge problem? Do not do this... We want to have the latest ubi images, should not affect arm.